routes/associations.js

```diff
@@ -22,10 +22,26 @@ router.put('/', [
 	check('name')
 		.not().isEmpty()
 		.withMessage("Ce champ ne peut pas rester vide."), 
+			// name
+	check('birthdate')
+	.not().isEmpty()
+	.withMessage("Ce champ ne peut pas rester vide."), 
```

Comment:
Il faut respecter l'indentation notamment pour ".not().isEmpty()" & ".withMessage()" par rapport à "check()"